src/ArmTemplates/Extractor/Utilities/Authentication.cs

```diff
@@ -9,28 +9,29 @@ namespace Microsoft.Azure.Management.ApiManagement.ArmTemplates.Extractor.Utilit
 {
     public class Authentication
     {
-        internal static bool isTokenValid = false;
-        public static DateTime start = DateTime.Now;
-        internal static string internalAzToken;
-        internal static string internalAzSubscriptionId;
+        internal static bool IsTokenValid = false;
+        public static DateTime Start = DateTime.Now;
+        internal static string InternalAzToken;
```

Comment:
Can you change these into properties?

Reply:
Investigated here: they are only used inside of this class, so I made that as private fields :)